tools/star.wdl

```diff
@@ -555,7 +555,7 @@ task alignment {
         Int outSAMflagOR = 0
         Int outSAMflagAND = 65535
         Int outFilterMultimapScoreRange = 1
-        Int outFilterMultimapNmax = 20
+        Int outFilterMultimapNmax = 50
```

Comment:
I'm not thrilled about this one: the default is actually quite permissible, so thinking that there is going to be a valid range for 21-49 different loci _per read_ is unlikely IMO. That said, I don't have a sense of how many reads are mapped to `mapped to too many loci` today.

Personally, I think keeping the default selected by Alex and then overriding this for Arriba is also the way to go here.

Reply:
For `tMN` with our existing pipeline, there are an average of `260382` reads per sample that are mapped to too many loci. That's between `0.11%` and `0.96%` of reads.

Reply:
If I expand to all available STAR logs (`/research_jude/rgs01_jude/groups/zhanggrp/projects/SJCloud/common/plaid/work/*/*/TRANSCRIPTOME/*/*/results/*.final.out`), then the average is `338989` with a median of `145909`.

Reply:
Okay, so it disproportionately affects some samples. Interesting.